test/test_files/dml_node/set/set_read.test

```diff
@@ -63,3 +63,40 @@ Alice2021-06-30
 Bob1950-05-14
 Carol2000-01-01
 Dan
+
+-CASE SetMultipleReadTest
```

Comment:
You probably don't need this test because it's not related to the change you made in this PR.